source/NVDAObjects/window/excel.py

```diff
@@ -954,6 +934,7 @@ def script_changeSelection(self,gesture):
 			if oldSelection.parent==newSelection.parent:
 				newSelection.parent=oldSelection.parent
 			eventHandler.executeEvent('gainFocus',newSelection)
+	script_changeSelection.canPropagate = True

```

Comment:
this also should not be there - previously this was done when decorating and should remain that way